src/modules/LanItems.rb

```diff
@@ -1311,25 +1339,26 @@ def BuildLanOverview
         note = ""
         bullets = []
         ifcfg_name = LanItems.Items[key]["ifcfg"] || ""
+        ifcfg_type = NetworkInterfaces.GetType(ifcfg_name)
 
-        LanItems.type = NetworkInterfaces.GetType(ifcfg_name)
         if !ifcfg_name.empty?
           ifcfg_conf = GetDeviceMap(key)
           ifcfg_desc = ifcfg_conf["NAME"]
           descr = ifcfg_desc if !ifcfg_desc.nil? && !ifcfg_desc.empty?
-          descr = CheckEmptyName(LanItems.type, descr)
+          descr = CheckEmptyName(ifcfg_type, descr)
           ip = DeviceProtocol(ifcfg_conf)
-          status = DeviceStatus(
-            LanItems.type,
-            ifcfg_name,
-            ifcfg_conf
-          )
+          status = DeviceStatus(ifcfg_type, ifcfg_name, ifcfg_conf)
 
           bullets << _("Device Name: %s") % ifcfg_name
           bullets += startmode_overview(key)
+
+          if bridge_index[ifcfg_name]
+            ip = bridge_ip(ip).empty? ? "NONE" : bridge_ip(ip)
```

Comment:
if the interface is edited before removing from bridge, then yast should remove it from bridge. Otherwise you could put system into inconsistent / incorrect state.